pokemongo_bot/cell_workers/follow_path.py

```diff
@@ -139,8 +140,8 @@ def work(self):
             'position_update',
             formatted="Walking from {last_position} to {current_position}, distance left: ({distance} {distance_unit}) ..",
             data={
-                'last_position': (last_lat, last_lng, last_alt),
-                'current_position': (lat, lng, alt),
+                'last_position': (last_lat, last_lng, 0),
+                'current_position': (lat, lng, 0),
```

Comment:
0 is not good. we have random altitude for now, please help to check if sending 0 to server.